lib/package.rb

```diff
@@ -154,4 +154,12 @@ def self.system(*args)
     exitstatus = $?.exitstatus
     raise InstallError.new("`#{args.join(" ")}` exited with #{exitstatus}") unless exitstatus == 0
   end
+  def self.make_install (destdir = CREW_DEST_PREFIX, opt = '')
+    @warning = 'libtool: warning: remember to run '
+        @out = "#{CREW_BREW_DIR}/libtool.txt"
+
+    system "make DESTDIR=#{destdir} install #{opt} 2>&1 |\
+            tee #{@out} |\
+            grep -v \"#{@warning}\""
+  end
```

Comment:
This is just removing the message.  This does nothing to solve the problem.

Reply:
This will put the original output to the libtool.txt file (the warning will save and execute in `crew`) while deleting the warning on the screen.